layers/place/city.sql

```diff
@@ -31,7 +31,7 @@ FROM (
          FROM osm_city_point
          WHERE geometry && bbox
            AND ((zoom_level = 2 AND "rank" = 1)
-             OR (zoom_level BETWEEN 3 AND 7 AND "rank" <= zoom_level + 1)
+            OR (zoom_level BETWEEN 3 AND 7 AND "rank" <= zoom_level + 1)
```

Comment:
Please leave the existing whitespace/formatting, again to make rebase easier in the future.

Reply:
I've updated this